steps/monkop-bitrise-android/2.0.0/step.yml

```diff
@@ -0,0 +1,49 @@
+title: Monkop.com Android Testing Cloud.
+summary: Test your Android app using Monkop. Real performance analysis using real
+  devices.
+description: |-
+  Use this step to test your deployed APK on Monkop.
+
+  Monkop enable to run smoke tests on several real devices with no officer. Get detailed information about the native app’s performance.
+
+  If you don't have a Monkop account, register for free [here](https://console.monkop.com/register.xhtml) and get your API KEY!
+  The API_KEY value can be found on "Add-ons" section under "API Key" option.
+website: https://github.com/MonkopApps/bitrise-monkop-android
+source_code_url: https://github.com/MonkopApps/bitrise-monkop-android
+support_url: https://github.com/MonkopApps/bitrise-monkop-android/issues
+published_at: 2016-10-31T15:19:23.567698906-03:00
+source:
+  git: https://github.com/MonkopApps/bitrise-monkop-android.git
+  commit: dfcceb8c2a1554ef89867794fb75cf6b754e0d28
+host_os_tags:
+- ubuntu
+- osx-10.10
+project_type_tags:
+- android
+type_tags:
+- script
```

Comment:
I don't think this step has anything to do with the `script` tag, as `script` steps let the user to write scripts